Utilities/wearable-packager/marketplace-packager.js

```diff
@@ -0,0 +1,144 @@
+            prevID = 0;
+        }
+    }
+
+    var baseUserdata = {
+        "Attachment": {
+            "action": "attach",
+            "joint": "Hips",
+            "attached" : false,
+            "options": {
+                "translation": {
+                    "x": 0,
+                    "y": 0,
+                    "z": 0
+                },
+                "scale": 1
+            }
+        },
+        "grabbableKey": {
+            "cloneable": false,
+            "grabbable": true
+        }
+    };
+
+    var exportProperties = {
+        type: "Model",
+        clientOnly: 1,
+        parentID: "{00000000-0000-0000-0000-000000000001}",
+        owningAvatarID: "{00000000-0000-0000-0000-000000000000}",
+        visible: 1,
+        shapeType: "box",
+        collidesWith: "",
+        collisionMask: 0
+    };
+
+    var button = tablet.addButton({
+        text: APP_NAME,
+        icon: APP_ICON
+    });
+
+    
+    function maybeExited() {
+        Entities.clickReleaseOnEntity.disconnect(handleMousePress);
+        Entities.hoverLeaveEntity.disconnect(handleMouseLeave);
+        tablet.screenChanged.disconnect(maybeExited);
+    }
+
+    function clicked(){
+        tablet.gotoWebScreen(APP_URL);
+        Entities.clickReleaseOnEntity.connect(handleMousePress);
+        Entities.hoverLeaveEntity.connect(handleMouseLeave);
+        Script.setTimeout(function(){
+            tablet.screenChanged.connect(maybeExited); 
+        }, 2000);
+    }
+    button.clicked.connect(clicked);
+
+    function onFileSaveChanged(filename){
+        Window.saveFileChanged.disconnect(onFileSaveChanged);
+        if (filename !== "") {
+            var success = Clipboard.exportEntities(filename, [entityIDToExport]);
+            if (!success) {
+                // No luck, failed
+            }
+        }
+        Entities.deleteEntity(entityIDToExport);                        
+    }
+
+    function onWebEventReceived(event){
+        if (typeof(event) === "string") {
+            event = JSON.parse(event);
+        }
+        if (event.type === "submit") {
+            var entityID = event.entityID;
+            var joint = event.joint;
+
+            var newExportProperties = exportProperties;
+            var newUserDataProperties = baseUserdata;
+    
+            var properties = Entities.getEntityProperties(entityID, ['modelURL', 'dimensions', 'script']);
+
+            newUserDataProperties["Attachment"].joint = joint;
+            
+            newExportProperties.modelURL = properties.modelURL;
+            newExportProperties.dimensions = properties.dimensions;
+            newExportProperties.parentJointIndex = MyAvatar.jointNames.indexOf(joint);
+            newExportProperties.script = properties.script;
+            newExportProperties.userData = JSON.stringify(newUserDataProperties);
+
+            entityIDToExport = Entities.addEntity(newExportProperties, 1);
```

Comment:
1 -> true     // to make it more clear for devs that read the script that it actually is a boolean